### ISSUE TITLE
Build it to something more useful

### ISSUE THREAD

**Issue by Dizcordum** [Kattoor/nw-model-miner#6]
Im not good at code, and i dont understand half of what writed in readme, nor i dont understand what i need to do to extract-convert models from paks to gltf format
Can you make it a gui programm? or at least add more steps in readme info, what exactly to do to get models?

**Comment by SphinxRa40** [comment]
agreed and would like more info also, GitHub is great but some things i have trouble with it to figure out :/
